src/components/ContactUs.jsx

```diff
@@ -61,9 +54,6 @@ function ContactUS() {
           paddingY: '50px',
```

Comment:
change all px values to rem. px values does not work well with displays of different sizes